apps/codeforafrica/src/components/ImpactCardList/ImpactCardList.js

```diff
@@ -0,0 +1,73 @@
+import { Section } from "@commons-ui/core";
+import { Link } from "@commons-ui/next";
+import { Grid, Box, Button, Typography } from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+import ImpactCard from "../ImpactCard/ImpactCard";
+
+const ImpactCardList = React.forwardRef(function ImpactCardList(props, ref) {
+  const { initiatives, action, title } = props;
+  return (
+    <Box
+      sx={{
+        backgroundColor: "background.main",
+      }}
+    >
+      <Section
+        sx={{
+          px: { xs: 2.5, sm: 2.5, md: 0 },
+          py: { xs: 5.25, sm: 5.25, md: 4.25, lg: 4.25, xl: 12.75 },
+        }}
+        ref={ref}
+      >
+        {title && (
+          <Typography
+            sx={{
+              marginBottom: "40px",
+              marginTop: { sm: 3.25, md: 3.75, lg: 0 },
+            }}
+            variant="h4"
+          >
+            {title}
+          </Typography>
+        )}
```

Comment:
```suggestion
        <RichTypography
          sx={{
            mb: { xs: 10, md: 5, lg: 7.55 },
            mt: { xs: 2.25, sm: 0, md: 4, lg: 0 },
          }}
          variant="h4"
        >
          {title}
        </RichTypography>
        )}
```

1. `RichTypography` already handles checking for `null`/`undefined`.
2. Spacing is based off of about page where there is a title